packages/cli/package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "compile-ts": "rm -rf lib && tsc",
-    "prepare": "npm run compile-ts",
+    "prepare": "npm run compile-ts && chmod 755 ./lib/bin/zos-cli.js",
```

Comment:
shouldn't this be part of the `compile-ts` script? I guess every time we compile TS we want that to happen, not only during installation right?

Reply:
Strictly speaking, `compile-ts` should only **compile** (transpile :confused:) ts code to js code. IMO, `prepare` should manage both compilation plus any other task that leaves the  `cli` **prepared** to run.